test/sheet/drink_sheet.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from file import MemoryFile
+from file.sheet import DrinkSheet
+from user.drinks import DrinkFactory
+from user.drinks.types import Coffee
+
+
+class TestDrinkSheet(TestCase):
+    def test_constructor(self):
+        coffee = Coffee(id_number=1, name='Coffee', cost=1.0)
+        drink_factory = DrinkFactory
+
+        mocked_get = MagicMock(return_value=coffee)
+        drink_factory.get = mocked_get
+
+        data = {'id_number': 1, 'name': 'Coffee', 'type': 1, 'cost': 1.0}
```

Comment:
``
```suggestion
        data = {'id_number': 1, 
                     'name': 'Coffee', 
                     'type': 1, 
                     'cost': 1.0}
```
Makes it easier to view the dictionary

Reply:
The actual data here doesn't matter and should never need to be changed so I think splitting the dict across 6 lines instead of just 1 hurts actually readability here 

The line length here is also still within the 79 character limit and well within the 99 character extended limit too so there's no need to change it

As a note: when we split the literal definition of a dict, list, set, tuple, etc. the opening and closing brackets should be on their own lines:
```
data = {
    'id_number': 1, 
    'name': 'Coffee', 
    'type': 1, 
    'cost': 1.0
}
```